packages/patternfly-4/react-core/src/components/AboutModal/AboutModal.d.ts

```diff
@@ -8,8 +8,7 @@ export interface AboutModalProps extends HTMLProps<HTMLDivElement> {
   trademark?: string;
   brandImageSrc: string;
   brandImageAlt: string;
-  logoImageSrc?: string;
```

Comment:
@tlabaj @dgutride I'm okay with removing these without saying this a breaking change because they never did anything.  If a consumer used them nothing would happen they aren't implemented in the component.

Reply:
When initially implemented, the image was not part of css.  So these props did something at one point, but with the current implementation, they are not doing anything.